README.md

```diff
@@ -154,10 +154,39 @@ sdk "iphoneos9.0"
 
 clean true
 
+export_options ({
```

Comment:
Unfortunately, without the `{` I get different an error.

In case of:
```ruby
export_options (
  method: "ad-hoc"
)
```

I got:
```
/Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane_core-0.36.1/lib/fastlane_core/configuration/configuration_file.rb:28:in `rescue in initialize': [!] Syntax error in your configuration file './fastlane/Gymfile' on line 6: (eval):6: syntax error, unexpected tLABEL (RuntimeError)
  method: "ad-hoc"
         ^
(eval):7: syntax error, unexpected ')', expecting end-of-input
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane_core-0.36.1/lib/fastlane_core/configuration/configuration_file.rb:22:in `initialize'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane_core-0.36.1/lib/fastlane_core/configuration/configuration.rb:146:in `new'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane_core-0.36.1/lib/fastlane_core/configuration/configuration.rb:146:in `load_configuration_file'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/gym-1.5.0/lib/gym/detect_values.rb:10:in `set_additional_default_values'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/gym-1.5.0/lib/gym.rb:25:in `config='
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/gym-1.5.0/lib/gym/manager.rb:4:in `work'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane-1.57.0/lib/fastlane/actions/gym.rb:22:in `run'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane-1.57.0/lib/fastlane/runner.rb:143:in `block (2 levels) in execute_action'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane-1.57.0/lib/fastlane/actions/actions_helper.rb:35:in `execute_action'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane-1.57.0/lib/fastlane/runner.rb:129:in `block in execute_action'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane-1.57.0/lib/fastlane/runner.rb:128:in `chdir'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane-1.57.0/lib/fastlane/runner.rb:128:in `execute_action'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane-1.57.0/lib/fastlane/fast_file.rb:149:in `method_missing'
	from Fastfile:105:in `block (2 levels) in parsing_binding'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane-1.57.0/lib/fastlane/lane.rb:36:in `call'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane-1.57.0/lib/fastlane/runner.rb:110:in `try_switch_to_lane'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane-1.57.0/lib/fastlane/fast_file.rb:141:in `rescue in method_missing'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane-1.57.0/lib/fastlane/fast_file.rb:136:in `method_missing'
	from Fastfile:172:in `block (2 levels) in parsing_binding'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane-1.57.0/lib/fastlane/lane.rb:36:in `call'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane-1.57.0/lib/fastlane/runner.rb:52:in `block in execute'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane-1.57.0/lib/fastlane/runner.rb:46:in `chdir'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane-1.57.0/lib/fastlane/runner.rb:46:in `execute'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane-1.57.0/lib/fastlane/lane_manager.rb:46:in `cruise_lane'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane-1.57.0/lib/fastlane/command_line_handler.rb:30:in `handle'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane-1.57.0/bin/fastlane:38:in `block (2 levels) in run'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/commander-4.3.5/lib/commander/command.rb:178:in `call'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/commander-4.3.5/lib/commander/command.rb:153:in `run'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/commander-4.3.5/lib/commander/runner.rb:428:in `run_active_command'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane_core-0.36.1/lib/fastlane_core/ui/fastlane_runner.rb:23:in `run!'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/commander-4.3.5/lib/commander/delegates.rb:15:in `run!'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane-1.57.0/bin/fastlane:150:in `run'
	from /Users/user/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/fastlane-1.57.0/bin/fastlane:156:in `<top (required)>'
	from /Users/user/.rbenv/versions/2.3.0/bin/fastlane:23:in `load'
	from /Users/user/.rbenv/versions/2.3.0/bin/fastlane:23:in `<main>'
```

There is no problem when we have `{` inside `(`. So for now I'll not remove this.